src/components/ExpenseForm/ChooseCategory/ChooseCategory.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+ import * as S from './style';
+
+const ChooseCategory = params => {
+  function handleChangeCategory(e) {
+    params.onCategoryChange(e.label, e.value)
```

Comment:
you can pass entire payload and split it in parent as an result of flattening two `Choose...` components into one.

Reply:
Fix into one component